### ISSUE TITLE
Is this library maintained?

### ISSUE THREAD

**Issue by JayWalker512** [Pithikos/C-Thread-Pool#78]
There are numerous years-old pull requests, the library appears to have been left to rot. That would be a shame because I haven't been able to find any other good pure-C threadpool libraries.

**Comment by Pithikos** [comment]
Hi @JayWalker512. In the past I would go through the PRs in chunks every few months whenever I found time. Unfortunately I have been quite busy with my life the last period so didn't have the chance to put more time into this. I understand your concern which is also why I built the library in the first place. However since I don't use C anymore on a daily basis and due to having to deal with other things, I can't tell if I will be able to go through the PRs in the short-term at least.

I'm happy to give access to people to maintain the project, assuming they share the same vision for keeping the solution elegant and minimal.

Cheers

**Comment by gonzus** [comment]
I volunteer to help merging some of the PRs, if you offer still stands. In fact, I will send at least one of my own, just for docs, because I see the code changes I wanted to make have already been included in other PRs.

**Comment by Pithikos** [comment]
@gonzus yes sure, that would be great! Just let me update the CI, since I noticed it's broken, and I will add you.

**Comment by gonzus** [comment]
Hey, got the invite, already approved two PRs. I didn't want to approve a pending PR of my own (https://github.com/Pithikos/C-Thread-Pool/pull/91/), thought it would be bad form -- if you are OK with that, I can approve it.

**Comment by dunglas** [comment]
Hi @Pithikos,

As we rely heavily on your lib (thanks for your work!) for [FrankenPHP](https://frankenphp.dev), I would also be happy to get the commit right to merge the fixes we need :)

Cheers,

**Comment by Pithikos** [comment]
I merged a few things and with Github Actions now (thanks @dunglas) things should be easier. Feel free to nudge me for anything to improve collaboration. Have a good 2024 :cake:

**Comment by ati13703** [comment]
Haha! That latest merge resolved my problem, on my workload the threads got stuck without doing any work sometimes. I ended up rewriting thpool.c and went back here to see how I could contribute. No need anymore, nice work.
